robotino_bringup/robots/tuw-robotino2/config/hand.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-# Vienna hand config parameters 
+# KCL/Freiburg hand config parameters 
```

Comment:
is this the  config for the Freiburgs hand? if yes, please move the file to robotino_bringup/robots/alufr-robotino/config/

Reply:
done. should be vienna hand config parameters. Already fix.